From 0bf530aac152630500939db31f98d933158fdabd Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Tue, 26 Feb 2019 14:27:49 +0800
Subject: [PATCH] Revert "meson: print EFI CC configuration nicely"

This reverts commit c512dfb9ac948ddb1ced0dab07b9dac88b198293.

This patch is here because we haven't figured out how to pass
parameter to efi-cc if it's an array in systemd-boot recipe.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 70fb218..30df834 100644
--- a/meson.build
+++ b/meson.build
@@ -3138,7 +3138,7 @@ if conf.get('ENABLE_EFI') == 1
         if have_gnu_efi
                 status += [
                         'EFI machine type:                  @0@'.format(EFI_MACHINE_TYPE_NAME),
-                        'EFI CC                             @0@'.format(' '.join(efi_cc)),
+                        'EFI CC                             @0@'.format(efi_cc),
                         'EFI lib directory:                 @0@'.format(efi_libdir),
                         'EFI lds directory:                 @0@'.format(efi_ldsdir),
                         'EFI include directory:             @0@'.format(efi_incdir)]
-- 
2.7.4

